db-server/index.js

```diff
@@ -3,10 +3,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var restify = require('restify')
-var safeJsonFormatter = require('restify-safe-json-formatter')
 var bufferize = require('./lib/bufferize')
 var version = require('../package.json').version
 var errors = require('./lib/error')
+const safeJsonFormatter = require('./lib/safeJsonFormatter')
```

Comment:
Since the `safeJsonFormatter` lib wasn't too big, opted to pull this into our repo. The new version of restify doesn't accept a `cb` for formatters.

Reply:
Just adding context, this is vendored in from https://github.com/g-k/restify-safe-json-formatter/blob/master/index.js and is already Mozilla code written by g-k (so we'll pulling in code Mozilla already owns, not some third-party dep)